t/request.t

```diff
@@ -571,3 +571,91 @@ qr/\[TRACE   \d+ content_by_lua\(nginx\.conf:\d+\):4 loop\]/
 bad argument type
 stitch
 
+
+
+=== TEST 17: ngx.req.get_post_args
+--- http_config eval: $::HttpConfig
+--- config
+    location = /t {
+        set $foo hello;
+        content_by_lua_block {
+            local ffi = require "ffi"
```

Comment:
Yeah, looks like this is a legacy holdover from a number of other tests (I copypasta'd the test structure from the get_args() tests); i'll remove this one and push up a separate PR to get rid of this cruft in the existing tests.